index.d.ts

```diff
@@ -382,8 +551,8 @@ export type PluginTuple<PluginParameters extends any[] = any[]> = [
  *   Plugin settings.
  */
 export type Pluggable<PluginParameters extends any[] = any[]> =
-  | PluginTuple<PluginParameters>
-  | Plugin<PluginParameters>
+  | PluginTuple<PluginParameters, any, any>
+  | Plugin<PluginParameters, any, any>
```

Comment:
would it make sense/add value to carry the generics through this type as well, rather then `any`s?

Reply:
yes, I’ll try!